mobile/mobile-core/android-process-service.ts

```diff
@@ -75,21 +77,65 @@ export class AndroidProcessService implements Mobile.IAndroidProcessService {
 				// Process information will look like this (without the columns names):
 				// USER     PID   PPID  VSIZE   RSS   WCHAN    PC         NAME
 				// u0_a63   25512 1334  1519560 96040 ffffffff f76a8f75 S com.telerik.appbuildertabstest
-				let processIdInformation: string = adb.executeShellCommand(["ps", "|grep", "-a", "--text", processId]).wait();
-
-				return _.last(processIdInformation.trim().split(/[ \t]/));
+				let processIdInformation: string = adb.executeShellCommand(["ps", "|grep", processId]).wait();
+				let appIdentifier = _.last(processIdInformation.trim().split(/[ \t]/));
+
+				return {
+					packageId: appIdentifier,
+					framework: this.$projectConstants.TARGET_FRAMEWORK_IDENTIFIERS.Cordova,
+					title: this.getPageTitleFromWebView(adb, deviceIdentifier, appIdentifier).wait()
+				};
 			}
 
 			// Search for appIdentifier (@<appIdentifier>_devtools_remote).
 			let chromeAppIdentifierRegExp = /@(.+)_devtools_remote\s?/g;
+			let chromeAppIdentifierMatches = chromeAppIdentifierRegExp.exec(information);
+
+			if (chromeAppIdentifierMatches && chromeAppIdentifierMatches.length > 0) {
+				let appIdentifier = chromeAppIdentifierMatches[1];
+
+				return {
```

Comment:
This `return` statement appears similar to the previous `cordova`-related `return` statement. Perhaps they could be unified into one